### DISCUSSION TITLE
「徒手抓取」化學分子 VR技術助新藥開發

### DISCUSSION THREAD

**Discussion by Lai27330** [1101-GS4719-C/Python-Community-GS4719-C#476] [General]
利用有趣的VR 技術 結合遊戲又可以透過簡單也不易出錯的方法來研發出新型藥物 也可以在醫療界相關的課程中作為高還原度教材供學生們精進所學 是個對現代醫療醫學頗有貢獻的新技術 
以下為網址https://news.ltn.com.tw/news/world/breakingnews/2833584

**Comment by chupochun** [comment]
雖然感覺比不上真實的教材，不過結合VR技術，似乎多了些許的新鮮感
